src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1179,11 +1180,18 @@ public override void WriteDebugLine(string message)
             }
             else
             {
-                // NTRAID#Windows OS Bugs-1061752-2004/12/15-sburns should read a skin setting here...
-                WriteLine(
-                    DebugForegroundColor,
-                    DebugBackgroundColor,
-                    StringUtil.Format(ConsoleHostUserInterfaceStrings.DebugFormatString, message));
+                if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
```

Comment:
Maybe we should check whether VT sequence is supported too. When this feature is enabled but a legacy console is in use, the code flow should take the `else` block.

Reply:
I see, the utils helpers check if VT is supported, but in your scenario the fg and bg colors wouldn't be there, will add it